backend/api/src/main/java/com/virtuslab/gitmachete/backend/hooks/BaseHookExecutor.java

```diff
@@ -1,8 +1,9 @@
-package com.virtuslab.gitmachete.backend.impl.hooks;
+package com.virtuslab.gitmachete.backend.hooks;
```

Comment:
This PR moves some (albeit not much) of the hooks implementation into `backend:api` (which shouldn't contain the implementation)... which I'm not very happy with, but I've considered the alternatives and it ended up being the best option currently